esphome/components/nextion/display.py

```diff
@@ -6,6 +6,8 @@
 
 DEPENDENCIES = ['uart']
 AUTO_LOAD = ['binary_sensor']
+AUTO_LOAD = ['switch']
+AUTO_LOAD = ['sensor']
```

Comment:
This would need to be
```suggestion
AUTO_LOAD = ['switch', 'sensor']
```
with the current syntax you're just redefining the variable